tests/reference/run_dbg-test_raise_01-dfd86ca.stderr

```diff
@@ -1,3 +1,5 @@
+  File "tests/runtime_errors/test_raise_01.py", line 1
+    def test():
   File "tests/runtime_errors/test_raise_01.py", line 4
     test()
   File "tests/runtime_errors/test_raise_01.py", line 2
```

Comment:
The new location info (`main_program` Program node info) is useless for LPython, but it is required for LFortran.

Reply:
Right, I knew this would be an issue. It's fine for now, we can refine this as we go. Possibly not doing so many indirections in LPython.